pkg/iam/policy/admin-action.go

```diff
@@ -117,6 +117,17 @@ const (
 	// GetBucketTargetAction - allow getting bucket targets
 	GetBucketTargetAction = "admin:GetBucketTarget"
 
+	// Transition Storage-Class admin Actions
+
+	// AddStorageClassAction - allow adding new transition storage-classes
+	AddStorageClassAction = "admin:AddStorageClass"
+	// EditStorageClassAction - allow editing transition storage-classes
+	EditStorageClassAction = "admin:EditStorageClass"
+	// RemoveStorageClassAction - allow removing transition storage-classes
```

Comment:
it seems like there is no benefit in having Add/Edit permission granularity. Can it instead just be SetStorageClass permission that applies to both Add and Edit commands